updatezinfo.py

```diff
@@ -17,9 +18,19 @@ def main():
 
     if not os.path.isfile(metadata['tzdata_file']):
         print("Downloading tz file from iana")
-        request.urlretrieve(os.path.join(metadata['releases_url'],
-                                         metadata['tzdata_file']),
-                            metadata['tzdata_file'])
+        for x in range(1, 6):
+            try:
+                request.urlretrieve(os.path.join(metadata['releases_url'],
+                                                 metadata['tzdata_file']),
+                                    metadata['tzdata_file'])
+                break
+            except Exception:
```

Comment:
What exception should this be catching, shouldn't you just catch that one? Some quick googling suggests `URLError`?

*Thanks for that Hypothesis article btw! :)*